src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -1,59 +1,89 @@
-import { Typography } from "@material-ui/core";
-// import { format, parseISO } from "date-fns";
+import { IconButton, TableCell, Tooltip } from "@material-ui/core";
+import { format, parseISO } from "date-fns";
 import React, { useState, useEffect } from "react";
 import { fetchCompanyOffers } from "../../../../services/companyOffersService";
 import ControlledSortableSelectableTable from "../../../../utils/Table/ControlledSortableSelectableTable";
 import FilterableTable from "../../../../utils/Table/FilterableTable";
-import { generateTableCellFromField } from "../../../../utils/Table/utils";
+import { alphabeticalSorter, generateTableCellFromField } from "../../../../utils/Table/utils";
 import { columns } from "./CompanyOffersManagementSchema";
 import PropTypes from "prop-types";
-import { format, parseISO } from "date-fns";
 import useSession from "../../../../hooks/useSession";
-
-const CompanyOffersNonFullfilledRequest = ({ isLoading, error }) => {
-    if (isLoading) {
-        return (
-            <h1>Loading...</h1>
-        );
-    } else if (error) {
-        return (
-            <h1>
-                {error?.msg ?? error}
-            </h1>
-        );
-    } else {
-        return null;
-    }
-};
-
-const generateRow = ({ title, location, publishDate, publishEndDate }) => ({
+import { OfferTitleFilter, PublishDateFilter, PublishEndDateFilter, LocationFilter } from "../Filters/index";
+import { Edit as EditIcon } from "@material-ui/icons";
+import { Link } from "react-router-dom";
+import { addSnackbar } from "../../../../actions/notificationActions";
+import { connect } from "react-redux";
+// import { RowActions } from "../Actions";
+
+const generateRow = ({ title, location, description, publishDate, publishEndDate,
+    ownerName, _id }) => ({ // Check if it is company.name or companyName
     fields: {
-        title: { value: title, align: "left" },
+        title: { value: title, align: "left", customComponent:
+    <Link to={`/offer/${_id}`} style={{ color: "initial", textDecoration: "underline", textUnderlineOffset: 2 }}>
+        {title}
+    </Link>,
```

Comment:
This is a bit ugly/verbose. Instead, when defining RowContent, check if the field has a prop such as `linkDestination`. If it does, render a link instead, with the style you are defining here. Then you just need to pass `linkDestination: `/offer/${_id}`` here, which is cleaner.

Also, instead of using `styles` directly, use the className+useStyles pattern as is being done throughout the application, which is more efficient